mortar/src/main/java/mortar/bundler/BundleService.java

```diff
@@ -84,6 +84,7 @@ void init() {
       }
 
       @Override public void onExitScope() {
+        if (runner.rootBundle != null) runner.rootBundle.remove(runner.bundleKey(scope));
         for (Bundler b : bundlers) b.onExitScope();
```

Comment:
This one liner style (for both lines) really is confusing to me. Brackets don't hurt and make things much clearer to me. Also, b => bundler.